Dockerfile

```diff
@@ -1,47 +1,38 @@
-FROM node:18-alpine AS base
-
```

Comment:
we can remove all extra infra like this for now: Dockerfiles, nginx config, etc.